packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -159,7 +124,12 @@ const DailyVideoApiAdapter = (): VideoApiAdapter => {
     const body = await translateEvent(event);
     const dailyEvent = await postToDailyAPI(endpoint, body).then(dailyReturnTypeSchema.parse);
     const meetingToken = await postToDailyAPI("/meeting-tokens", {
-      properties: { room_name: dailyEvent.name, exp: dailyEvent.config.exp, is_owner: true },
+      properties: {
+        room_name: dailyEvent.name,
+        exp: dailyEvent.config.exp,
+        is_owner: true,
+        enable_recording_ui: false,
```

Comment:
set enable_recording_ui: false to hide default recording button because we want are using customTrayButtons